modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/edit_form_instance_record.jsp

```diff
@@ -29,15 +29,16 @@ String title = ParamUtil.getString(request, "title");
 renderResponse.setTitle(GetterUtil.get(title, LanguageUtil.get(request, "view-form")));
 %>
 
-<div class="portlet-forms">
-	<div class="ddm-form-basic-info ddm-form-success-page">
-		<div class="container-fluid-1280">
-			<h1 class="ddm-form-name"><%= ddmFormAdminDisplayContext.getFormName() %></h1>
+<div class="container-fluid-1280 ddm-form-builder-app editing-form-entry">
```

Comment:
@marcosapmf, where is the `</div>`? Did you try to move this `div` to include the form name and description elements? I'm not sure if I got what's the point of this modification, can you please clarify?

Reply:
@natocesarrego sorry, this `<div>` is duplicated, it should be caused by rebase that I did. I'm going to fix it :)